crates/pera/src/keytool.rs

```diff
@@ -600,7 +601,27 @@ impl KeyToolCommand {
                         peer_id: None,
                     })
                 }
+                SignatureScheme::ClassGroups => {
```

Comment:
Again I don't understand why are you treating class groups as a signature scheme, it is not